src/cdk/a11y/live-announcer/live-announcer.spec.ts

```diff
@@ -92,6 +92,19 @@ describe('LiveAnnouncer', () => {
       expect(ariaLiveElement.textContent).toBe('Hello there');
     }));
 
+    it('should synchroneously clear a message even if not yet displayed' , fakeAsync(() => {
```

Comment:
synchroneously -> synchronously. Also the spacing before the comma is a little weird.

Reply:
Done.